www/js/angular-react-helper.jsx

```diff
@@ -38,18 +38,18 @@ export function angularize(component, modulePath) {
     .component(nameCamelCase, makeComponentProps(component));
 }
 
-const roots = new Map();
 export function makeComponentProps(Component) {
   const propTypes = Component.propTypes || {};
   return {
     bindings: toBindings(propTypes),
-    controller: /*@ngInject*/ function($element) {
-      let root = roots.get($element[0]);
-      if (!root) {
-        root = createRoot($element[0]);
-        roots.set($element[0], root);
-      }
+    controller: function($element) {
+      /* TODO: once the inf scroll list is converted to React and no longer uses
+        collection-repeat, we can just set the root here one time
+        and will not have to reassign it in $onChanges. */
+      /* Until then, React will complain everytime we reassign an element's root */
+      let root;
```

Comment:
Just to clarify, this reverts https://github.com/JGreenlee/e-mission-phone/commit/39ddf90840833397dbb8fc7c289895c8709cad2d and we are just going to live with the errors 😄

Reply:
Correct. I don't see a way around the errors for now.

Hopefully this will only be for a few weeks, until the list itself is rewritten in React.